UDF_Swift/AppDelegate.swift

```diff
@@ -36,12 +36,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window = UIWindow(frame: UIScreen.main.bounds)
         window?.rootViewController = UIViewController()
 
-        let mainRoutable = HomeViewRoutable(window: window!)
-        router = Router(store: projectsStore, rootRoutable: mainRoutable) {
+        let mainRoutable = HomeViewRouter(window: window!)
```

Comment:
maybe rename the var as well? ->`mainRouter`

Reply:
Yes, good point